packages/shared/src/settings/background.ts

```diff
@@ -10,24 +10,31 @@ import { BACKGROUND_COLOR_DEFAULT_VALUE } from './defaultValues';
  * @param options.id Custom suffix for the setting ids
  * @param options.defaultValue Default value for the background switch
  * @param options.defaultColor Default value for the background color
+ * @param options.hasDefaultColor Whether the background color should have a default value
+ * @param options.switchLabel Label for the background switch
  * @returns {SettingBlock} Returns border settings
```

Comment:
```suggestion
 * @returns {SettingBlock} Returns background settings
```
Probably i missed that one